cumulus/parachains/runtimes/testing/penpal/src/xcm_config.rs

```diff
@@ -275,25 +277,41 @@ where
 	}
 }
 
+// This asset can be added to AH as Asset and reserved transfer between Penpal and AH
+pub const RESERVABLE_ASSET_ID: u32 = 1;
 // This asset can be added to AH as ForeignAsset and teleported between Penpal and AH
 pub const TELEPORTABLE_ASSET_ID: u32 = 2;
+
+pub const ASSETS_PALLET_ID: u8 = 50;
+pub const ASSET_HUB_ID: u32 = 1000;
+
 parameter_types! {
 	/// The location that this chain recognizes as the Relay network's Asset Hub.
-	pub SystemAssetHubLocation: Location = Location::new(1, [Parachain(1000)]);
+	pub SystemAssetHubLocation: Location = Location::new(1, [Parachain(ASSET_HUB_ID)]);
+	pub SystemAssetHubLocationV3: xcm::v3::Location = xcm::v3::Location::new(1, [xcm::v3::Junction::Parachain(ASSET_HUB_ID)]);
+	pub RelayLocationV3: xcm::v3::Location = xcm::v3::Location::parent();
 	// ALWAYS ensure that the index in PalletInstance stays up-to-date with
 	// the Relay Chain's Asset Hub's Assets pallet index
 	pub SystemAssetHubAssetsPalletLocation: Location =
-		Location::new(1, [Parachain(1000), PalletInstance(50)]);
+		Location::new(1, [Parachain(ASSET_HUB_ID), PalletInstance(ASSETS_PALLET_ID)]);
 	pub AssetsPalletLocation: Location =
-		Location::new(0, [PalletInstance(50)]);
+		Location::new(0, [PalletInstance(ASSETS_PALLET_ID)]);
 	pub CheckingAccount: AccountId = PolkadotXcm::check_account();
 	pub LocalTeleportableToAssetHub: Location = Location::new(
 		0,
-		[PalletInstance(50), GeneralIndex(TELEPORTABLE_ASSET_ID.into())]
+		[PalletInstance(ASSETS_PALLET_ID), GeneralIndex(TELEPORTABLE_ASSET_ID.into())]
 	);
 	pub LocalTeleportableToAssetHubV3: xcm::v3::Location = xcm::v3::Location::new(
```

Comment:
please, remove `LocalTeleportableToAssetHubV3` and `LocalReservableFromAssetHubV3`, because whoever wants to use them, has possibility to use `LocalTeleportableToAssetHub` and `LocalReservableFromAssetHub` and downcast to whatever XCM version is needed, this is really not necessary to keep it here. Let's just use `xcm::latest::*` and whoever can downcast to desired XCM version.

I also removed usage of this stuff in the fellows: https://github.com/polkadot-fellows/runtimes/pull/187/commits/ea3ebe4d0f98b25d67c49772a2a727e3f4f675e0